scripts/test_docker_local.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+
+set -ex
+
+cli=./tigris
+export cli
+
+CN=test-docker-local
+
+curl -sSL https://tigris.dev/cli-linux | tar -xz -C .
```

Comment:
Consider using a variable for the hardcoded URL

Reply:
Good idea.